main/2024/CG-03-26.md

```diff
@@ -0,0 +1,39 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the March 26th video call of WebAssembly's Community Group
+
+- **Where**: zoom.us
+- **When**: March 26th, 5pm-6pm UTC (March 26th, 9am-10am Pacific Time)
+- **Location**: *link on calendar invite*
+
+### Registration
+
+None required if you've attended before. Send an email to the acting [WebAssembly CG chair](mailto:webassembly-cg-chair@chromium.org)
+to sign up if it's your first time. The meeting is open to CG members only.
+
+## Logistics
+
+The meeting will be on a zoom.us video conference.
+Installation is required, see the calendar invite.
+
+## Agenda items
+
+1. Opening, welcome and roll call
+    1. Opening of the meeting
+    1. Introduction of attendees
+1. Find volunteers for note taking (acting chair to volunteer)
+1. Proposals and discussions
+    1. Discuss adding table64 to memory64 proposal: https://github.com/WebAssembly/memory64/issues/46
```

Comment:
Sorry I missed this the first time around, can you suggest a timebox?

Reply:
Done